nodeup/pkg/model/kube_proxy.go

```diff
@@ -61,6 +61,14 @@ func (b *KubeProxyBuilder) Build(c *fi.ModelBuilderContext) error {
 		}
 	}
 
+	if b.ConfigurationMode == "Warming" {
+		pullTask := &nodetasks.PullImageTask{
+			Name:    kubeProxyImage(b.NodeupModelContext),
+			Runtime: b.Cluster.Spec.ContainerRuntime,
+		}
+		c.AddTask(pullTask)
+	}
```

Comment:
It would be simpler to wrap this up in a receiver off of `NodeupModelContext`. It would be even simpler still to push the `ConfigurationMode` test down into the task (renaming it `WarmImageTask`.)